test/functional/pdo_sqlsrv/pdo_connection_resiliency_prepare_transact.phpt

```diff
@@ -5,7 +5,8 @@ Prepare a statement, break the connection, and execute the statement. Then
 test transactions by breaking the connection before beginning a transaction
 and in the middle of the transaction. The latter case should fail.
 --SKIPIF--
-<?php require('skipif_protocol_not_tcp.inc');
+<?php require('skipif_mid-refactor.inc');
+      require('skipif_protocol_not_tcp.inc');
       require('skipif_version_less_than_2k14.inc');  ?>
```

Comment:
same here... otherwise, this test will be skipped even when running against sql server 2014